### PR TITLE
Feature/mert

### PR DESCRIPTION
**Pull request by mertgokbayrak**
* Third commit
* test1.py conflict fixed

### CODE REVIEW

File: test1.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 print("Hi")
-print()
-print("Bye")
\ No newline at end of file
+print("aaaaa")
```
Comment (Amirkia1998) [inline test1.py:2]: Can you change understandable keyword?

**Review by Amirkia1998 — CHANGES_REQUESTED** [review]
can you fix the review notes ?

File: test1.py (hunk 1 above)
Comment (mertgokbayrak) [reply test1.py]: fixed

**Review by Amirkia1998 — APPROVED** [review]
thanks @mertgokbayrak

**Pull request merged**